addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4831,7 +4831,31 @@ msgctxt "#2104"
 msgid "Check the log for more information."
 msgstr ""
 
-#empty strings from id 2105 to 2202
+#. Heading text for yes/no dialog to ask for add-on install to get his font file
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2105"
+msgid "Missing font language support"
+msgstr ""
+
+#. Used to show notification about automatic language add-on installation, for here the used text
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2106"
+msgid "Do you want to install the add-on to support characters?[CR][CR]Add-on: {0:s} ({1:s})[CR]Character Typ: \"{2:s}\""
+msgstr ""
+
+# Notification message if add-on installation about font failed.
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2107"
+msgid "Installation of the add-on \"{0:s}\" to support the written language \"{1:s}\" failed."
```

Comment:
Do we distinguish between written languages and spoken ones elsewhere in the GUI? This is the first time I'm seeing "written" describing a language.